src/controllers/analysis.js

```diff
@@ -0,0 +1,61 @@
+
+const formidable = require('formidable');
+const fs = require('fs');
+const nodemailer = require('nodemailer');
+const solutions = [];
+const solutinA1 = ' you have to do something for the entrance';
+
+
```

Comment:
remove EXTRA LINES :dagger:

Reply:
oops! I had changed it wallah! but I didn't pushed it!
I pushed before working anything so that Sallam can work on it!